<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChipQueryWidget</class>
 <widget class="QWidget" name="ChipQueryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>芯片测试数据查询系统</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>数据查询</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>批次ID</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>芯片UID</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>日期范围</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>至</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QDateEdit" name="endDateEdit"/>
        </item>
        <item row="1" column="1" colspan="3">
         <widget class="QLineEdit" name="lotIdLineEdit"/>
        </item>
        <item row="2" column="1" colspan="3">
         <widget class="QLineEdit" name="uidLineEdit"/>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="startDateEdit"/>
        </item>
        <item row="3" column="1" colspan="3">
         <widget class="QLineEdit" name="markingNoLineEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>打标号</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="resetButton">
          <property name="text">
           <string>重置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="searchButton">
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>查询结果</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="resultCountLabel">
          <property name="text">
           <string>查询结果: 0 条记录</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>每页显示:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="pageSizeComboBox">
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>25</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>100</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
